updated for Blender 2.8

doakey3 opened the pull request:
* Minor changes made to the code to bring it up to Blender version 2.8
* I put the code into a __init__.py file so users will install the add-on using the downloaded .zip which is more consistent with other newer add-ons
* I updated the README to describe installation and usage

doakey3 committed:
updated for Blender 2.8

boombatower commented on __init__.py:
```diff
@@ -122,64 +123,104 @@ def draw(self, context):
+    
     # add the frame change handler
-    bpy.app.handlers.scene_update_pre.append(typewriter_text_update_frame)
+    bpy.app.handlers.frame_change_pre.append(typewriter_text_update_frame)
```
Is the change from `scene_update_pre` to `frame_change_pre` and the change from `register_module` to `register_class` the only actual functional changes? It seems like virtually everything is style changes and README changes.

doakey3 replied:
Yes that about sums it up.

Commit:
Fix animation render with keyframed typewriter properties

We need to explicitly evaluate the object as described here:
https://wiki.blender.org/wiki/Reference/Release_Notes/2.81/Python_API#Handlers
in order to get all current value of any fcurve-animated properties.

Commit:
add gitignore

doakey3 committed:
Merge pull request #1 from gandalf3/master

 Fix animation render with keyframed typewriter properties
